google-cloud-storage/test/helper.rb

```diff
@@ -114,7 +116,13 @@ def billing_hash(requester_pays)
     { "requesterPays" => requester_pays} unless requester_pays.nil?
   end
 
-  def random_file_hash bucket=random_bucket_name, name=random_file_path, generation="1234567890", kms_key_name="path/to/encryption_key_name", custom_time: nil
+  def file_retention_hash(retention_params)
```

Comment:
[Optional Nit] I'd recommend specifying the method declaration as `def file_retention_hash(mode: nil, retain_until_time: nil)`. This ensures the users of this helper method wouldn't need to scan through the method body to know what keys the hash needs to contain.

Reply:
This is for a test helper. The actual code already has this mentioned (in file.rb)